### ISSUE TITLE
SEO suggestion: post blogs on medium

### ISSUE THREAD

**Issue by ZooeyMiller** [dwyl/dwyl-site#357]
I know that we are building blogs into the website so it may not be blogs strictly under dwyl's name, but for SEO purposes, having lots of links to a site from another high quality site (e.g. Medium  for blogs) boosts SEO really nicely. 

If, for example we set aside time for dwylers to post personal (of course web/digital product development related) blogs on Medium where they linked to the dwyl site, it could effect SEO positively. Thus, it could be worth considering. 

Finding ways to make sure we're linked by high quality sites in general would be a good thing to work towards, but it's more than something that can strictly be made into a PR on this issue.

**Comment by markwilliamfirth** [comment]
Linked to:
https://github.com/dwyl/hq/issues/265


@ZooeyMiller - this has been discussed a number of times (I am actually in favour of Medium) but @iteles is concerned that Medium will cause SEO issues (see https://github.com/dwyl/hq/issues/265#issuecomment-324573829)

Can you explain why it is beneficial?

**Comment by ZooeyMiller** [comment]
My understanding is that in modern SEO having high quality backlinks (links to our business's site from another high quality site) is one of the best ways to improve SEO. More information on [what a high quality backlink is can be found in this article](https://www.seomark.co.uk/high-quality-backlinks/).

I don't fully understand the mention of ["zero SEO benefit"](https://github.com/dwyl/hq/issues/265#issuecomment-283110876) from the linked issue. I don't think that having blogs on Medium should _replace_ our blogging platform (we would get SEO benefits from the content of said blogs, and hopefully increased traffic to the site due to said blogs). Instead if we encouraged/incentivised/made time for dwylers to write their own articles about digital product development on Medium, wherein they link to the dwyl site, we could perhaps benefit from both mediums. We should definitely not duplicate content, as this can actually negatively effect our SEO.

It is true, as mentioned in the article I linked, that the easier a backlink is to get then the less value it has, and a blog on medium definitely falls closer to easy than it does to hard. However if that link generates traffic, and has text that we wish to optimise for then my understanding is that it will help with SEO.
